samples/net/sockets/echo_async/README.rst

```diff
@@ -21,7 +20,7 @@ Requirements
 ************
 
 - :ref:`networking_with_qemu`
-- or, a board with hardware networking
+- or, a board with hardware networking (including 6LoWPAN)
```

Comment:
By adding text in parens I meant that this sample was actually tested on 6LoWPAN hardware. (Which is by definition IPv6 only, not supporting IPv4. Before I tested it, I wasn't sure if the sample would work out of the box in such case, and after I tested it, I wanted to give users a hint that it's supported, so for example, if it doesn't, it's not the sample limitation, but some other issue. (Note that another sample mentioned in the doc, sockets/echo, doesn't run on 6LoWPAN)).